src/pyodide/python-entrypoint-helper.ts

```diff
@@ -104,6 +104,9 @@ async function setupPatches(pyodide: Pyodide): Promise<void> {
     if (TRANSITIVE_REQUIREMENTS.has('fastapi')) {
       await injectSitePackagesModule(pyodide, 'asgi', 'asgi');
     }
+
+    // TODO: ideally we shouldn't need to inject this permanently.
+    await injectSitePackagesModule(pyodide, 'introspection', 'introspection');
```

Comment:
@hoodmane what's the best API to use here to avoid allowing the user to import this module?